cms/envs/test.py

```diff
@@ -132,6 +132,7 @@
 # This hack disables migrations during tests. We want to create tables directly from the models for speed.
 # See https://groups.google.com/d/msg/django-developers/PWPj3etj3-U/kCl6pMsQYYoJ.
 MIGRATION_MODULES = {app: "app.migrations_not_used_in_tests" for app in INSTALLED_APPS}
+MIGRATION_MODULES['credentials'] = 'app.migrations_not_used_in_tests'
```

Comment:
You solved the problem for credentials. What about the other apps? Again, isn't `credentials` in `INSTALLED_APPS`?